src/app/engine/self-host/page.mdx

```diff
@@ -19,6 +19,7 @@ dashboard](https://thirdweb.com/dashboard/engine).
 - [Docker](https://docs.docker.com/get-docker/)
 - A thirdweb secret key from the [API Keys page](https://thirdweb.com/dashboard/settings/api-keys)
 - PostgresDB (version 14+)
+- Redis (version 7.2.4+)
```

Comment:
https://docs.bullmq.io/bull/install suggests
>  Bull requires a Redis version greater than or equal to `2.8.18`

But most new users prob will just take the latest version anyway.